specification/v1/core/schema/trqp_response_schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "$id": "trqp-response",
+  "title": "TRQP Response",
+  "type": "object",
+  "required": [
+    "entity_id",
+    "assertion_id",
+    "authority_id",
+    "time_evaluated",
+    "assertion_verified"
+  ],
+  "properties": {
+    "entity_id": {
+        "type": "string",
+        "description": "TODO"
+    },
+    "authority_id": {
+        "type": "string",
+        "description": "The id of the Authority entity"
+    }, 
+    "assertion_id": {
+        "type": "string",
+        "description": "TODO"
+    },
+    "assertion_verified": {
+        "type" : "boolean",
+        "description": "True if the assertion has been verified, false otherwise."
+    },
+    "time_requested": {
+        "type": "string",
+        "format": "date-time",
+        "description": "the server time that was requested (may be blank)."
+    },
+    "time_evaluated": {
+        "type": "string",
+        "format": "date-time",
+        "description": "the server time that was used in the query."
+    },
+    "message": {
+        "type": "string"
+    },
+    "context": {
+        "type": "object",
+        "description": "the context object that was supplied for the",
+        "properties": {
+            "time": {
+                "type": "string",
+                "description": "Time for query to use. If blank, uses current server time. RFC3339 required, using only Z (+00:00) offset.",
+                "example": 
+                [ 
+                    {
+                    "time": "1985-04-12T23:20:50.52Z"
+                    }
+                ]
+            }
+        },
+        "additionalProperties": {
+            "type": "string"
```

Comment:
additional properties can be anything. I believe should be set toward true

"additionalProperties": true

https://json-schema.org/understanding-json-schema/reference/object#additionalproperties